app/assets/stylesheets/components/projects/form.scss

```diff
@@ -0,0 +1,21 @@
+.project-form {
+  margin-bottom: 0;
+  padding: 20px 60px;
+
+  &__title {
+    margin-bottom: 70px;
+  }
+
+  &__button {
+    width: 100%;
+
+    @media (min-width: 500px) {
+      width: 325px;
+    }
+  }
+
+  &__submit-btn {
```

Comment:
I've been playing with the styles of this button on staging 😄 

The styles on this class don't seem to have any effect. Is this needed?